absl/strings/string_view.cc

```diff
@@ -64,7 +64,7 @@ std::ostream& operator<<(std::ostream& o, string_view piece) {
     size_t rpad = 0;
     if (static_cast<size_t>(o.width()) > piece.size()) {
       size_t pad = static_cast<size_t>(o.width()) - piece.size();
-      if ((o.flags() & o.adjustfield) == o.left) {
+      if ((o.flags() & std::ostream::adjustfield) == std::ostream::left) {
```

Comment:
This change doesn't seem consistent with the rest of the changes in this change. I would have to ask that we avoid the churn for this one.